sentiment_analysis/scripts/train.py

```diff
@@ -0,0 +1,136 @@
+import json
+import time
+from pathlib import Path
+from typing import Optional
+
+import numpy as np
+import tensorflow as tf
+from dotenv import load_dotenv
+from pydantic import BaseSettings
+from sklearn.model_selection import train_test_split
+from transformers import DistilBertTokenizer, TFDistilBertForSequenceClassification
+
+GPU_AVAILABLE = tf.test.is_gpu_available()
+
+load_dotenv(".env")  # need to run script from root of project to read this
+
+
+class EnvVars(BaseSettings):
+    """Pydantic data class to hold environment variables"""
+
+    TRAIN_DATA_PATH: Path
+    EPOCHS: int = 1
+    BATCH_SIZE: int = 64 if GPU_AVAILABLE else 1
+    VAL_BATCH_SIZE: int = 16 if GPU_AVAILABLE else 1
+    SAVE_MODEL_DIR: Optional[Path] = None
```

Comment:
Not for this PR but we should move all this stuf into a config file (json/yaml or similar)

Reply:
Sure lets discuss that, I was using CLI args so far but saw my colleague use environment variables and pydantic classes a lot so thought I will try that